scripts/ic-update/metadata.json

```diff
@@ -2,9 +2,9 @@
     "formatVersion": 1,
     "components": [
         {
-            "id": "h3ulcb-4x2g-ab-1.0-cluster",
+            "id": "rcar-multinode-zephyr-1.0-cluster",
```

Comment:
Hi @iusyk, we need to add the '-spider' part based on the latest discussion.

The final name should be 'rcar-multinode-zephyr-1.0-spider-cluster'.

Please, change it.

Reply:
Done.